api/inventory/v1/agents.proto

```diff
@@ -723,6 +706,57 @@ message AddAgentResponse {
   int32 table_count = 20;
 }
 
+message ChangeAgentRequest {
+  oneof agent {
+    // ChangePMMAgentParams pmm_agent = 1;
```

Comment:
let's uncomment it and keep it empty for now

Reply:
Decided to leave for now.